sct.py

```diff
@@ -487,85 +486,81 @@ def list_resources(ctx, user, test_id, get_all, get_all_running, verbose):
         click.secho("Nothing found for selected filters in Docker!", fg="yellow")
 
 
-@cli.command('list-ami-versions', help='list Amazon Scylla formal AMI versions')
-@click.option('-r', '--region',
-              type=CloudRegion(cloud_provider="aws"),
-              default='eu-west-1',
-              help="a region to look for AMIs (default: eu-west-1)")
-@click.option('-a', '--arch',
-              type=click.Choice(AwsArchType.__args__),
-              default='x86_64',
-              help="architecture of the AMI (default: x86_64)")
-def list_ami_versions(region: str, arch: AwsArchType):
-    add_file_logger()
-
-    tbl = PrettyTable(field_names=["Name", "ImageId", "CreationDate"], align="l")
-    for ami in get_scylla_ami_versions(region_name=region, arch=arch):
-        tbl.add_row([ami.name, ami.image_id, ami.creation_date])
-    click.echo(tbl.get_string(title="Scylla AMI versions"))
-
-
-@cli.command('list-ami-branch', help="""list Amazon Scylla branched AMI versions
-    \n\n[VERSION] is a branch version to look for, ex. 'branch-2019.1:latest', 'branch-3.1:all'""")
+@cli.command('list-images', help="List machine images")
+@click.option('-b', '--backend',
+              type=click.Choice(["aws", "gce", "azure"]),
+              help="Cloud backend to query images for.")
+@click.option('-br', '--branch',
+              type=str,
+              default="master:latest",
+              help="Branch to query images for. Defaults to 'master:latest'")
+@click.option('-v', '--version',
+              type=str,
+              is_flag=True,
```

Comment:
how do you send a specific version if this is a flag? IIUC a flag is a `boolean`

Reply:
The original command did not filter on a version argument, so it made sense to make it a flag. That being said I changed the command somewhat to allow filtering by versions.